src/crawler_script.sh

```diff
@@ -0,0 +1,3 @@
+curl -v http://localhost:5000/runcrawler -o outfile
+
+echo 'i runnnnn' >> logs.txt
```

Comment:
Maybe let's put something more meaningful here 😂 
Can be also useful for when we set it up on google cloud to see logs